off-pm-collection.json

```diff
@@ -1,201 +1,13 @@
 {
 	"info": {
-		"_postman_id": "ec40dfb2-9041-4f7d-b930-0320f424e5e2",
+		"_postman_id": "35da6b52-6273-4213-abea-34ca58d7c3d3",
 		"name": "Open Food Facts API Documentation",
 		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json"
 	},
 	"item": [
 		{
 			"name": "1. General Information",
 			"item": [
```

Comment:
```suggestion
			"item": [
				{
					"name": "1.2 Authentication",
					"item": [
						{
							"name": "Login",
							"request": {
								"method": "POST",
								"header": [],
								"body": {
									"mode": "formdata",
									"formdata": [
										{
											"key": "user_id",
											"value": "username or email",
											"type": "text"
										},
										{
											"key": "password",
											"value": "password",
											"type": "text"
										}
									]
								},
								"url": {
									"raw": "{{PROTOCOL}}://world.{{DOMAIN_NAME}}/cgi/auth.pl",
									"protocol": "{{PROTOCOL}}",
									"host": [
										"world",
										"{{DOMAIN_NAME}}"
									],
									"path": [
										"cgi",
										"auth.pl"
									]
								}
							},
							"response": [
								{
									"name": "Correct credentials",
									"originalRequest": {
										"method": "POST",
										"header": [],
										"body": {
											"mode": "formdata",
											"formdata": [
												{
													"key": "user_id",
													"value": "correct username or email",
													"type": "text"
												},
												{
													"key": "password",
													"value": "correct password",
													"type": "text"
												}
											]
										},
										"url": {
											"raw": "{{PROTOCOL}}://world.{{DOMAIN_NAME}}/cgi/auth.pl",
											"protocol": "{{PROTOCOL}}",
											"host": [
												"world",
												"{{DOMAIN_NAME}}"
											],
											"path": [
												"cgi",
												"auth.pl"
											]
										}
									},
									"status": "OK",
									"code": 200,
									"_postman_previewlanguage": "html",
									"header": [
										{
											"key": "Server",
											"value": "nginx/1.10.3"
										},
										{
											"key": "Date",
											"value": "Sat, 11 Dec 2021 12:40:40 GMT"
										},
										{
											"key": "Content-Type",
											"value": "text/html; charset=ISO-8859-1"
										},
										{
											"key": "Transfer-Encoding",
											"value": "chunked"
										},
										{
											"key": "Connection",
											"value": "keep-alive"
										},
										{
											"key": "Vary",
											"value": "Accept-Encoding"
										},
										{
											"key": "Content-Encoding",
											"value": "gzip"
										},
										{
											"key": "Strict-Transport-Security",
											"value": "max-age=63072000"
										},
										{
											"key": "X-Frame-Options",
											"value": "DENY"
										},
										{
											"key": "X-Content-Type-Options",
											"value": "nosniff"
										}
									],
									"cookie": [],
									"body": null
								},
								{
									"name": "Wrong credentials",
									"originalRequest": {
										"method": "POST",
										"header": [],
										"body": {
											"mode": "formdata",
											"formdata": [
												{
													"key": "user_id",
													"value": "wrong username or email",
													"type": "text"
												},
												{
													"key": "password",
													"value": "wrong password",
													"type": "text"
												}
											]
										},
										"url": {
											"raw": "{{PROTOCOL}}://world.{{DOMAIN_NAME}}/cgi/auth.pl",
											"protocol": "{{PROTOCOL}}",
											"host": [
												"world",
												"{{DOMAIN_NAME}}"
											],
											"path": [
												"cgi",
												"auth.pl"
											]
										}
									},
									"status": "Forbidden",
									"code": 403,
									"_postman_previewlanguage": "html",
									"header": [
										{
											"key": "Server",
											"value": "nginx/1.10.3"
										},
										{
											"key": "Date",
											"value": "Sat, 11 Dec 2021 12:42:03 GMT"
										},
										{
											"key": "Content-Type",
											"value": "text/html; charset=ISO-8859-1"
										},
										{
											"key": "Transfer-Encoding",
											"value": "chunked"
										},
										{
											"key": "Connection",
											"value": "keep-alive"
										},
										{
											"key": "Content-Encoding",
											"value": "gzip"
										}
									],
									"cookie": [],
									"body": "<!DOCTYPE HTML PUBLIC \"-//IETF//DTD HTML 2.0//EN\">\n<html>\n    <head>\n        <title>403 Forbidden</title>\n    </head>\n    <body>\n        <h1>Forbidden</h1>\n        <p>You don't have permission to access /cgi/auth.pl\non this server.\n            <br />\n        </p>\n        <hr>\n        <address>Apache/2.4.25 (Debian) Server at world.openfoodfacts.org Port 80</address>\n    </body>\n</html>"
								}
							]
						}
					],
					"description": "#### Authentication\n\n##### READ and SEARCH operations\n\nNo authentication is required.\n\nAdd a User-Agent HTTP Header with the name of your app, the version, system and a url (if any), not to be blocked by mistake.\n\nFor example: User-Agent: NameOfYourApp - Android - Version 1.0 - [www.yourappwebsite.com](http://www.yourappwebsite.com)\n\n##### WRITE operations\n\nNo authentication is required for adding new products or adding images.\n\nBasic authentication is required for editing existing products. You can create a global account to let the users of your app contribute without having to create individual credentials in the Open Food Facts site. Then it would be good to provide some kind of hashed user identification so that we can in case of violations, trace actions back to specific accounts.\n\nOtherwise, you can create accunt, check credentials and even send password reset requests via the api."
				},
```